axlearn/experiments/text/gpt/c4_trainer.py

```diff
@@ -55,6 +55,11 @@
 )
 from axlearn.experiments.trainer_config_utils import TrainerConfigFn
 
+import jax
+import jax_neuronx
+from jax._src import compilation_cache 
+compilation_cache.set_cache_dir('./jax_cache')
```

Comment:
why in current working directory?

Reply:
fixed this...jax allows an env variable https://jax.readthedocs.io/en/latest/persistent_compilation_cache.html#caching-on-multiple-nodes